tests/simulator/config.py

```diff
@@ -1 +1,7 @@
 project_root = "@PHARE_PROJECT_DIR@"
+
+at = "@" # try to work without cmake config
+
+if project_root == f"{at}PHARE_PROJECT_DIR{at}":
```

Comment:
not related, but I keep making this change locally on every branch just to run tests from root and not the build dir